jaeger-core/src/test/java/io/jaegertracing/baggage/BaggageSetterTest.java

```diff
@@ -53,22 +54,22 @@ public void setUp() throws Exception {
     mgr = mock(DefaultBaggageRestrictionManager.class);
     setter = new BaggageSetter(mgr, metrics);
     tracer =
-        new Tracer.Builder(SERVICE)
+        new JaegerTracer.Builder(SERVICE)
             .withReporter(reporter)
             .withSampler(new ConstSampler(true))
             .withMetrics(metrics)
             .build();
-    span = (Span) tracer.buildSpan("some-operation").start();
+    jaegerSpan = (JaegerSpan) tracer.buildSpan("some-operation").start();
```

Comment:
is the cast still necessary?

Reply:
Yes, `tracer.buildSpan()` returns OT's SpanBuilder, which returns OT's Span for the `start()` method. It's basically the same as https://github.com/jaegertracing/jaeger-client-java/pull/454#discussion_r197764641